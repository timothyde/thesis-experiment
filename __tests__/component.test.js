import React from 'react';
import styled from 'styled-components';
import { shallow } from 'enzyme';
import 'jest-styled-components';

const Button = styled.button`
  color: #f00;
`;

test('it works', () => {
  const wrapper = shallow(<Button />);
  expect(wrapper).toMatchSnapshot();
});
