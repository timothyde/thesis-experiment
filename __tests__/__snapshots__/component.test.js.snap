// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  color: #f00;
}

<button
  className="c0"
/>
`;
